src/Pages/Dashboard.js

```diff
@@ -36,10 +40,10 @@ function DashboardPage(props) {
             <Watchlist setCurrentStock={setCurrentStock}/>
           </Grid>
           <Grid item xs={6} sm={4}>
-            <StockDisplay currentStock={currentStock} setCurrentStock={setCurrentStock}/>
+            <StockDisplay currentStock={currentStock} setCurrentStock={setCurrentStock} loggedUsername={props.loggedUsername} loggedIn={props.loggedIn}/>
           </Grid>
           <Grid item xs={6} sm={4}>
-
+            <Portfolio setCurrentStock={setCurrentStock} loggedUsername={props.loggedUsername} loggedIn={props.loggedIn} />
```

Comment:
Not sure that we need to keep track of whether or not the user is logged in here. The portfolio can only be displayed for the existing, current user.

Reply:
will change towards delivery! thanks!